components/formats-api/src/loci/formats/FormatWriter.java

```diff
@@ -330,6 +362,20 @@ public int setTileSizeY(int tileSize) throws FormatException {
     return height;
   }
 
+  /* @see IFormatWriter#getChunkSize() */
+  @Override
+  public int[] getChunkSize() throws FormatException {
+    int[] defaultChunkSize = {getTileSizeX(), getTileSizeY(), 1, 1, 1};
+    return defaultChunkSize;
+  }
+
+  /* @see IFormatWriter#setChunkSize(int[]) */
+  @Override
+  public int[] setChunkSize(int[] chunkSize) throws FormatException {
+    int[] returnChunkSize = {setTileSizeX(chunkSize[0]), setTileSizeY(chunkSize[1]), 1, 1, 1};
```

Comment:
Back to @melissalinkert's question about what would test this code, this could warn or error if a >2D chunk size were requested.